Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/ProgressIndicators/ProgressIndicatorLoadingBar.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Threading.Tasks;
+using TMPro;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions
+{
+    /// <summary>
+    /// Displays progress bar and (optionally) percentage text.
+    /// </summary>
+    [ExecuteAlways]
+    public class ProgressIndicatorLoadingBar : MonoBehaviour, IProgressIndicator
```

Comment:
@julenka question for you on what you think about experimental features as they relate to new UI/UX controls.

In particular, I'm kinda curious what your thoughts on if new features NEED tests and documentation before they graduate out of experimental.

Given how little time we have before we snap and how busy everyone else is, I want to avoid us adding new features that don't have tests and documentation collateral, where we need to support that API surface forever going forward. If we had tests/docs going in at the same time (or if this is literally the next thing that's going to be worked on, then I guess it's not a huge deal)